client/mass/store.js

```diff
@@ -334,6 +334,8 @@ TdbStore.prototype.actions = {
 				parent.lst[i] = replacementFilter
 			}
 		}
+		//In the profile runproteinpaint this function is passed to clear the local filters when the global filter changes
+		if (this.app.opts.app?.onFilterChange) this.app.opts.app.onFilterChange(this.state.plots)
```

Comment:
is this constructor option? i would like this documented somewhere, if not in a script then in client/mass/README

Reply:
It is used in the profile index and I add some comments there.

Reply:
There is no client/mass/README.md, should I add one?

Reply:
I added some comments in mass/app. Please check.